vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -0,0 +1,103 @@
+
+To employ Solr, the index and configuration files called the core, has to be created. This tutorial decribes the instructions for both the general and multitenant use cases. For a mulitenant application, a core has to be generated for the primary and each of the secondary tenants. For secondary tenants, see the [Multitenacy on Entando](./multitenancy-tutorial.md) tutorial.
+
+## Prerequisites
+* [A working instance of Entando](../../docs/getting-started/README.md). with the default Tomcat server image
+* Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md): `ent check-env develop`
+* [Helm](https://helm.sh/docs/intro/install/) to handle Solr installation
+
+## Install and Configure Solr 
+ 
+1. Install Solr Helm charts  
+```
+helm repo add apache-solr https://solr.apache.org/charts
+helm repo update
+```  
+For more information, see the [Solr Getting Started page](https://solr.apache.org/guide/7_3/solr-tutorial.html).
+
+2. Install the Solr and Zookeeper CRDs
+```
+kubectl create -f https://solr.apache.org/operator/downloads/crds/v0.5.0/all-with-dependencies.yaml
+```
+3. Install the Solr operator and Zookeeper Operator
+```
+helm install solr-operator apache-solr/solr-operator --version 0.5.0
+```
+4. Download the template `entando-solrCloud.yaml`
+
+<EntandoCode>curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/{{ $site.themeConfig.entando.fixpack.v72 }}/dist/ge-1-1-6/samples/entando-solrCloud.yaml"</EntandoCode>
+
+5. Replace the placeholders in `entando-solrCloud.yaml` with the appropriate values for your environment.
+   
+   | Placeholder | Description |
+   |:--|:-- |
+   | YOUR-HOST-NAME | The base host name of the application, e.g., your-domain.com |
+
+3. Apply the Solr application YAML
+```
+kubectl apply -f entando-solrCloud.yaml -n YOUR-NAMESPACE
+```
+4. To check that the service has started properly:
+```
+kubectl get solrclouds -w
+```
+You should see a response similar to this:
+```
+NAME   VERSION   TARGETVERSION   DESIREDNODES   NODES   READYNODES   UPTODATENODES   AGE
+solr   8                         3              3       3            3               79m
+```
+
+## Generate the Core
+
+1. Enable port-forwarding
+First you need to enable a port forwarding to the Solr common service using Kubectl
+```
+kubectl port-forward service/solr-solrcloud-common 5432:80
+```
+kubectl port-forward does not return. To continue, you need to open another terminal.
+
+2. Call the Solr API to generate the core
+
+   | Placeholder | Description |
+   |:--|:-- |
+   | YOUR-TENANT-ID | The identifier for the tenant, e.g., mysite1 |
+
+If you are building the primary tenant, then you have to use "entando" instead of "YOUR-TENANT-ID" for the collection name.
```

Comment:
We can simplify a bit by shelling into the solr-solrcloud-0 to execute this curl command. It feels simpler to me since you can do it in one shell, or in Rancher/OpenShift directly, but you need two for port-forwarding.

Reply:
I think both are quite simple since it requires a kubectl command and a curl command. I have no strong idea about which one is better though. 

The only thing  I thought was that using the port forwarding and the service cloud-common leverage Solr abstraction, we don't need to target a specific pod.

Reply:
Per our discussion yesterday, we'll keep the port-forwarding option in place